homeassistant/components/climate/nest.py

```diff
@@ -97,6 +100,16 @@ def __init__(self, structure, device, temp_unit):
         self._min_temperature = None
         self._max_temperature = None
 
+    @property
+    def should_poll(self):
+        """Do not need poll thanks using Nest streaming API."""
+        return False
+
+    async def async_nest_update_event_handler(self, event):
+        """Update device state."""
+        await self.async_device_update()
```

Comment:
```py
self.async_schedule_update_ha_state(True)
```

Reply:
After change to use dispatcher, can I use `awit self.async_update_ha_state(True)` here?
As my understanding, dispatcher will create a job for the callback, if I use `async_schedule_update_ha_state`, it will add another job, seems unnecessary

Reply:
Yes, that should also work. Then keep this method a coroutine.

Reply:
Done